documentation/sphinxext/docscrape.py

```diff
@@ -463,7 +463,7 @@ def __str__(self):
 class ClassDoc(NumpyDocString):
     def __init__(self,cls,modulename='',func_doc=FunctionDoc,doc=None):
         if not inspect.isclass(cls):
-            raise ValueError("Initialise using a class. Got %r" % cls)
```

Comment:
This isn't actually our code, although I'm not positive of either the canonical source, nor if this is still the way we should be using it